homeassistant/components/open_meteo/weather.py

```diff
@@ -78,3 +78,49 @@ def wind_bearing(self) -> float | str | None:
         if not self.coordinator.data.current_weather:
             return None
         return self.coordinator.data.current_weather.wind_direction
+
+    @property
+    def forecast(self) -> list[Forecast] | None:
+        """Return the forecast in native units."""
+        if self.coordinator.data.daily is None:
+            return None
+
+        forecasts: list[Forecast] = []
+        for index, time in enumerate(self.coordinator.data.daily.time):
+            forecast = Forecast(
+                datetime=time.isoformat(),
```

Comment:
Do timezones need to be accounted for here?

Reply:
It already has been accounted for, see the initialization in the unit file